_drafts/2017-10-13-rprofile-david-smith.md

```diff
@@ -0,0 +1,118 @@
+---
+name: rprofile-david-smith
+layout: post_discourse
+title: ".rprofile: David Smith"
+date: 2017-10-13
+authors:
+  - name: Kelly O'Briant
+    url: https://kellobri.github.io/
+tags:
+- R
+- community
+- interviews
+---
+
+<div class="alert alert-info" role="alert">
+In this occasional series, we interview someone using a loosely defined set of interview questions for the purpose of “Demystifying the Creative/Development Process of the R Community”. This interview was conducted and prepared by Kelly O'Briant as part of an rOpenSci unconf17 project.
+</div>
+
+<br>
+
+<img src="/assets/blog-images/2017-10-13-rprofile_david_smith/david-smith.jpg" alt="David Smith, R Community Lead at Microsoft" align="left" style="margin: 0px 20px", width="250px">
+
+_David Smith is a Blogger and Community Lead at Microsoft. I had the chance to interview David last May at rOpenSci unconf17. We spoke about his career, the process of working remote within a team, community development/outreach and his personal methods for discovering great content to share and write about._
+
+---
+
+<br>
+
+KO: What is your name, job title, and how long have you been using R?
+
+DS: My name is David Smith. I work at Microsoft and my self-imposed title is ‘R Community Lead’. I’ve been working with R specifically for about 10 years, but I’d been working with S since the early 90s.
+
+KO: How did you transition into using R?
+
+DS: I was using S for a long, long time, and I worked for the company that commercialized S, where I was a project manager for S-PLUS. And I got out of that company, and then worked for a startup in a different industry for a couple of years. But while I was there, the people that founded Revolution Analytics approached me because they were setting up a company to build a commercial business around R, and reached out to me because of my connections with the S community.
+
+KO: So you came to Microsoft through Revolution?
+
+DS: That’s correct. I was with Revolution Analytics and then Microsoft bought that company, so I’ve been with Microsoft since then.
+
+KO: How has that transition gone, is there a Revolution team inside of Microsoft, or has it become more spread out?
+
+DS: It’s been more spread out. It got split up into the engineering people and the marketing people, sales people all got distributed around. When I first went to Microsoft I started off in the engineering group doing product management. But I was also doing the community role, and it just wasn’t a very good fit just time-wise, between doing community stuff and doing code or product management. So then I switched to a different group called the Ecosystem team, and so now I’m 100% focused on community within a group that’s focused on the ecosystem in general.
+
+> <h3>The one piece of advice I could give anyone starting out in their careers is - write what you do, write it in public, and make it so that other people can reproduce what you did.</h3>
```

Comment:
@karthik  formatting intent here is as a pullquote that clearly stands out. I'm not sure if this `> <h3>text</h3>` approach will work

You suggested `<blockquote>text</blockquote>` but I don't know if the text will stand out enough and I'm not able to do local render right now to play with it.

We'll use this exact structure and formatting of at least two more blog posts (one per month) so I'd appreciate advice on getting this right.

Reply:
looks good to me, looks like this 
<img width="936" alt="screen shot 2017-10-12 at 1 13 20 pm" src="https://user-images.githubusercontent.com/577668/31520727-b18c2fd0-af5b-11e7-9797-c04ea283899e.png">

Reply:
I like that too